src/org/usfirst/frc/team2175/subsystem/drivetrain/DrivetrainSubsystem.java

```diff
@@ -64,17 +69,29 @@ public DrivetrainSubsystem() {
 
         robotDrive = new RobotDrive(leftMasterMotor, rightMasterMotor);
 
-        analogGyro = new AnalogGyro(
-                wiringProperties.getDrivetrainAnalogGyroDeviceNumber());
-
-        // navXGyro = new AHRS(SPI.Port.kMXP);
+        navXGyro = new AHRS(SPI.Port.kMXP);
         switchToPercentVbus();
+        // TODO Propertize
+        leftEncoder = new Encoder(0, 1, true);
+        leftEncoder.setDistancePerPulse(1);
+        rightEncoder = new Encoder(0, 1, true);
```

Comment:
@NoahConnors These port numbers can't possibly work. Either get it into the properties file (you probably have time) or find out what they actually are on the bot. At least make it so they don't overlap.